ui/app/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -70,8 +70,17 @@ export default function AwaitingSwap ({
 
   let feeinFiat
   if (usedQuote && tradeTxParams) {
-    const renderableGasFees = getRenderableGasFeesForQuote(usedQuote.gasEstimateWithRefund || usedQuote.averageGas, approveTxParams?.gas || '0x0', tradeTxParams.gasPrice, currentCurrency, conversionRate)
-    feeinFiat = renderableGasFees.feeinFiat?.slice(1)
+    const renderableNetworkFees = getRenderableNetworkFeesForQuote(
+      usedQuote.gasEstimateWithRefund || usedQuote.averageGas,
+      approveTxParams?.gas || '0x0',
+      tradeTxParams.gasPrice,
+      currentCurrency,
+      conversionRate,
+      tradeTxParams.value,
+      sourceTokenInfo?.symbol,
+      usedQuote.sourceAmount,
```

Comment:
This one at least is validated as a `truthyDigitString`, which I'm assuming means it's a decimal string. The `sourceAmount` passed into the API appears to be the "non-decimal-adjusted / GWEI" amount, so presumably that's what they pass back as well?

Reply:
Though weirdly, the defaults used in `quotesToRenderableData` for `sourceAmount` and `destinationAmount` are `number`s rather than strings.

Reply:
The issues raised in the above comments are addressed in https://github.com/MetaMask/metamask-extension/pull/9730